thewalrus/__init__.py

```diff
@@ -118,7 +118,7 @@
     interferometer,
     grad_hermite_multidimensional,
 )
-from ._permanent import perm, permanent_repeated
+from ._permanent import perm, permanent_repeated, brs, ubrs
```

Comment:
@jakeffbulmer : you probably want to add `brs` and `ubrs` into the `__all__` list in line 132

Reply:
and also into the autosummary in line 78. This is so that they are included when the documentation is generated.